### DISCUSSION TITLE
System Tray Icon Spacing?

### DISCUSSION THREAD

**Discussion by SearingEchoes** [valinet/ExplorerPatcher#2476] [Q&A]
I read in another thread that it's an advanced setting. But they didn't say how or where. I'd really like to make the spacing for my system tray icons back to how they were before the KB5022913 update for W10 that screwed it all up.

I'd like these to be closer together.
![2023-11-20_11-23-51](https://github.com/valinet/ExplorerPatcher/assets/843166/ac65fba7-5a27-4adb-931b-7a0d3a168dbb)

**Comment by Amrsatrio** [comment]
It's no longer possible after that update. Microsoft has removed the old tray icons code from the latest versions of the Windows 11 taskbar. If you want close icon spacings, then you have to use the Windows 10 taskbar.

<img width="193" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/3214d576-0000-4a6f-902b-c578b31758f0">

**Reply by SearingEchoes** [reply]
I'm a little unclear as to what you mean, I'm using ExplorerPatcher to have the Windows 10 taskbar. Are you saying I have to downgrade to Windows 10? I had the same problem in 10 as well before I upgraded.

**Reply by Amrsatrio** [reply]
Hasn't the spacing always been like this since early Win10 builds? In my reply, I meant the switch of the tray icons from legacy Win32 (despite the new taskbar) into XAML which has animations but hideously big spacing. Which happened in 22621.1344 or 1413 aka Moment 2.

**Reply by SearingEchoes** [reply]
No, it was a more recent thing, last year I think. I just want the ability to change it. I have to widen my taskbar by two pixels so the icons aren't in a single line.
![2023-11-20_16-13-27](https://github.com/valinet/ExplorerPatcher/assets/843166/543939c4-baa9-4c4b-8317-9ebb7e789306)

**Comment by pyrates999** [comment]
> I'm a little unclear as to what you mean, I'm using ExplorerPatcher to have the Windows 10 taskbar. Are you saying I have to downgrade to Windows 10? I had the same problem in 10 as well before I upgraded.

in EP properties application, you set the windows taskbar to be windows 10.

**Reply by SearingEchoes** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/843166/378cc637-21e5-46a0-adaf-039690e09085)
I already do looks like.

**Comment by PastelOrbit** [comment]
It may already be resolved, but if your display supports touch input, disabling the ‘HID-compliant Touch Screen’ in Device Manager and then restarting should solve the problem.
